mock/py/mockbuild/plugins/package_state.py

```diff
@@ -17,7 +17,7 @@
 
 # repoquery used
 repoquery_avail_opts = \
-    "--qf '%{name}-%{epoch}:%{version}-%{release}.%{arch} %{buildtime} %{size} %{pkgid} %{repoid}' '*'"
+    "--qf '%{name}-%{epoch}:%{version}-%{release}.%{arch} %{buildtime} %{size} %{pkgid} %{repoid}' --last '*'"
```

Comment:
I don't see `--last` in `dnf5-repoquery` manual page, nor in `repoquery` (dnf-utils).  Doesn't it actually limit the query output to a single package of each %{NAME}?